commit b29c65623f508082ded87af6f8d068ce8882f936
Author: Brad Hubbard <bhubbard@redhat.com>
Date:   Thu Sep 13 13:19:02 2018 +1000

    mgr: Change signature of PyString_AsString to match return
    
    PyUnicode_AsUTF8 now returns 'const char*'
    
    Fixes: http://tracker.ceph.com/issues/35984
    
    Signed-off-by: Brad Hubbard <bhubbard@redhat.com>

--- a/src/mgr/PythonCompat.h
+++ b/src/mgr/PythonCompat.h
@@ -13,7 +13,7 @@
 inline PyObject* PyString_FromString(const char *v) {
   return PyUnicode_FromFormat("%s", v);
 }
-inline char* PyString_AsString(PyObject *string) {
+inline const char* PyString_AsString(PyObject *string) {
   return PyUnicode_AsUTF8(string);
 }
 inline long PyInt_AsLong(PyObject *io) {
